arlo_server/models.py

```diff
@@ -1,8 +1,10 @@
 # type: ignore
-from arlo_server import db
 from sqlalchemy.orm import relationship, backref
 from typing import Union, List
 from enum import Enum
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
```

Comment:
This was the key change - not passing `app` into SQLAlchemy, and closing that loop later (see __init__.py)